cmd/snap/cmd_run_test.go

```diff
@@ -449,8 +450,8 @@ func (s *SnapSuite) TestSnapRunIsReexeced(c *check.C) {
 		readlink string
 		expected bool
 	}{
-		{filepath.Join(dirs.SnapMountDir, "/usr/lib/snapd/snapd"), true},
-		{"/usr/lib/snapd/snapd", false},
+		{filepath.Join(dirs.SnapMountDir, dirs.DistroLibExecDir, "snapd"), true},
```

Comment:
For now yes, but when we have switch to different core/base snaps it may not. Using dirs.CoreLibExecDir here now.